fem/libceed/ceed.cpp

```diff
@@ -139,6 +229,26 @@ void InitCeedTensorBasisAndRestriction(const mfem::FiniteElementSpace &fes,
                              tp_el_dof.GetData(), restr);
 }
 
+void InitCeedBasisAndRestriction(const mfem::FiniteElementSpace &fes,
+                                 const mfem::IntegrationRule &irm,
+                                 Ceed ceed, CeedBasis *basis,
+                                 CeedElemRestriction *restr)
+{
+   const bool tensor = dynamic_cast<const mfem::TensorBasisElement *>(fes.GetFE(
+                                                                         0)) ? true : false;
+   const mfem::IntegrationRule &ir = tensor ?
```

Comment:
The logic of this method would be easier to follow if the ternary was inside the `if`/`else` statement since they use the same conditional check